pyigm/surveys/igmsurvey.py

```diff
@@ -463,7 +463,7 @@ def fill_ions(self, use_Nfile=False, jfile=None, use_components=False,
             raise ValueError("Not sure how to load the ions")
 
     # Get ions
-    def ions(self, Zion, Ej=0., skip_null=True):
+    def ions(self, Zion, Ej=0., skip_null=True, pad_with_nulls=False):
```

Comment:
Note, I updated CgmSurvey to pass , e.g., 'pad_with_nulls' through in calls to functions that require this method (like ion_tbl()).